README.md

```diff
@@ -83,38 +83,16 @@ Troubleshooting help can be found [here](https://rapidsai.github.io/cuxfilter/in
 
 ## Installation
```

Comment:
I would add something like:
**NOTE:** cuxfilter is in ongoing development and the installation instructions will be updated in the near future.